packages/data-store/src/statusList/StatusListStore.ts

```diff
@@ -158,11 +144,37 @@ export class StatusListStore implements IStatusListStore {
     return !error
   }
 
+  async removeStatusListEntryByIndex(args: IGetStatusListEntryByIndexArgs): Promise<boolean> {
```

Comment:
```suggestion
  async removeStatusListEntryByIndex(args: IGetStatusListEntryByIndexArgs, error: boolean = false): Promise<boolean> {
```
than don't need to add a let on the body

Reply:
, error: boolean = false
This would break the API interface contract